### ISSUE TITLE
Android 14 (Api 34) support

### ISSUE THREAD

**Issue by Arattan** [exponea/exponea-android-sdk#70]
Hello,

Can I please request that the targetSDK is updated to Android Api 34? It seems that as your SDK currently targets Api 33 and [your `NotificationsPermissionReceiver`](https://github.com/exponea/exponea-android-sdk/blob/3.13.0/sdk/src/main/java/com/exponea/sdk/receiver/NotificationsPermissionReceiver.kt) does not [specify a export flag](https://developer.android.com/about/versions/14/behavior-changes-14#runtime-receivers-exported), according to the new behaviour changes, an exception is caused when we update our targetSDK. Here is the error which appears in the logs:

```
Exponea Safe Mode wrapper caught unhandled error
java.lang.SecurityException: One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts
```

Thanks in advance :)

Aman

**Comment by adam1929** [comment]
Thank you @Arattan for reporting. We have to upgrade targetSdkVersion to 34 and bring such a flag to receiver (among other changes to support Android 34).
As this issue has to be re-implemented in SDK, there so setup you could do in your app. Only way to "fix" this is to implement your NotificationPermission dialog.

**Comment by xcibik00** [comment]
Hi @Arattan , we already fixed it in 3.14.0 version
